eng/Signing.props

```diff
@@ -23,10 +23,19 @@
     <FileSignInfo Include="PackageWithFakeNativeDep.dll" CertificateName="None" />
     <FileSignInfo Include="dotnet-tool-with-output-name.dll" CertificateName="None" />
 
-    <!-- Third party DLLs used by tests -->
+    <!-- Third party DLLs used by tests-->
     <FileSignInfo Include="Castle.Core.dll" CertificateName="None" />
     <FileSignInfo Include="Moq.dll" CertificateName="None" />
 
+    <!-- Third party DLLs used in src projects not participating in source-build -->
```

Comment:
Source-build isn't a differentiator here for signing, maybe it's OK to not sign since this is a non-shipping tool?  If it's shipping, then we should sign with the 3rd party cert `$(ExternalCertificateId)` as is done for other 3rd party bits below.

Reply:
This dependency is used in product code, unlike the ones using `None` which are dependencies for test code. So it makes sense to use `$(ExternalCertificateId)` as you suggest.